### DISCUSSION TITLE
Apple Silicon Support

### DISCUSSION THREAD

**Discussion by Astilex5** [voxeltycoon/issues#830] [General]
I wanted to throw the idea of Apple Silicon support out there as Unity 2020.2 LTS is supposed to have export options for arm64 chips.

I feel this would future-proof the game for when Apple drops Rosetta 2 (Which is very likely to happen in the coming years).

**Comment by ndelta0** [comment]
Voxel Tycoon already uses Unity version 2020.3.7

**Reply by Astilex5** [reply]
That could give reason to release an Apple Silicon build. It would increase performance, I personally feel if the Devs need Beta Testers, Making a Beta Branch on Steam and let the people know on r/macgaming that testing help is needed would be a good idea.

I am sure people would gladly help with Testing (Myself included)

**Reply by ndelta0** [reply]
I don't know how Steam handles Apple Silicon version currently, so that may not be completely possible currently. Regarding the beta branch it's already there.

**Reply by Astilex5** [reply]
My assumption is one could replace the Intel Only Binary with a Universal 2 Binary. Unity has it as a build option (See 2nd Screenshot)

OpenTTD ships their macOS version of the game as such (See 1st Screenshot) 

So it's not so much managing two builds of the game. It's leveraging Universal 2 for Both x86_64 and ARM64. Given the OpenTTD devs has their U2 Build in the Steam Depot I assume steam will treat it like any other binary.

I should note I am a Web Developer. I am just a hobbyist when it comes to Unity. So forgive my limited knowledge.

![Screen Shot 2021-05-30 at 6 09 52 PM](https://user-images.githubusercontent.com/24304507/120122505-3175ba00-c177-11eb-862b-6ef01ae63f40.png)
![Screen Shot 2021-05-30 at 6 38 45 PM](https://user-images.githubusercontent.com/24304507/120122506-3175ba00-c177-11eb-90d8-cbab3b35890b.png)
